src/VisualStudio/IntegrationTest/TestUtilities/InProcess/Editor_InProc.cs

```diff
@@ -252,16 +252,12 @@ public Signature GetCurrentSignature()
         public bool IsCaretOnScreen()
             => ExecuteOnActiveView(view =>
             {
-                var editorPrimitivesFactoryService = GetComponentModelService<IEditorPrimitivesFactoryService>();
-                var viewPrimitivies = editorPrimitivesFactoryService.GetViewPrimitives(view);
+                var caret = view.Caret;
 
-                var advancedView = viewPrimitivies.View.AdvancedTextView;
-                var caret = advancedView.Caret;
-
-                return caret.Left >= advancedView.ViewportLeft
-                    && caret.Right <= advancedView.ViewportRight
-                    && caret.Top >= advancedView.ViewportTop
-                    && caret.Bottom <= advancedView.ViewportBottom;
+                return caret.Left >= view.ViewportLeft
+                    && caret.Right <= view.ViewportRight
+                    && caret.Top >= view.ViewportTop
+                    && caret.Bottom <= view.ViewportBottom;
```

Comment:
@DustinCampbell it looks like you added this code in aca6901c21aa8ddd8db31dc6c3a761255eb47fcf. Any idea why you were using the view primitives stuff vs. just the properties on the IWpfTextView?